### PR TITLE
Modularized Landing Page, CSV Additions

### PR DESCRIPTION
**Pull request by emmaleengo**
## Related Issues and Dependencies

…

## This introduces a breaking change

- [ ] Yes
- [ ] No

## This Pull Request implements

… Explain your changes.

### CODE REVIEW

**Comment by KPostOffice** [comment]
Is this in lieu of https://github.com/AICoE/Varangian/pull/44 ?

File: VarangianApplication/src/App.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import './app.scss';
+import { Content, Theme } from '@carbon/react';
+import Header from './components/Header';
+import { Route, Switch } from 'react-router-dom';
+import LandingPage from './content/LandingPage';
+import RepoPage from './content/RepoPage';
```
Comment (sahilsuneja1) [inline VarangianApplication/src/App.js:7]: Is this missing or TODO?

File: VarangianApplication/src/App.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import IssuePage from './content/IssuePage';
+import ParseCSVTest from './content/ParseCSVTest';
```
Comment (sahilsuneja1) [inline VarangianApplication/src/App.js:9]: Same as above comment; missing or TODO?

**Comment by sahilsuneja1** [comment]
> Is this in lieu of #44 ?
Right, @emmaleengo ?

**Pull request closed without merging**